src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -942,6 +955,31 @@ private function createOrders(
         $this->objectManager->flush();
     }
 
+    private function createPaidOrders($numberOfCustomers, $numberOfOrders, $total): void
+    {
+        $customers = $this->generateCustomers($numberOfCustomers);
+        $sampleProductVariant = $this->sharedStorage->get('variant');
+        $total = $this->getPriceFromString($total);
+
+        for ($i = 0; $i < $numberOfOrders; ++$i) {
+            $order = $this->createOrder($customers[random_int(0, $numberOfCustomers - 1)], '#' . uniqid());
+            $order->setState(OrderInterface::STATE_NEW); // Temporary, we should use checkout to place these orders.
```

Comment:
```suggestion
            $order->setState(OrderInterface::STATE_NEW);
```

Reply:
but it is true :( and we have this info in another function too

Reply:
Could you use state machine here?

Reply:
You may try to use OrderExampleFactory here.

Reply:
There is no reason in Heaven and Earth that should excuse using a `setState` method - and for sure it should be eradicated whenever it's used right now 🔥